### PR TITLE
Product history

### PR DESCRIPTION
**Pull request by janejuenyang**
Hi David - I've made a few tweaks to the Product History page based on what I found in the SvN blogs.

### CODE REVIEW

File: product-histories.md [hunk 1]
```diff
@@ -16,32 +18,40 @@ Fast forward to 2014. We'd been feeling pulled in too many directions for too lo
...
-
-## Know Your Company
+## Ta-da List
 
-In 2013, Basecamp created a software tool that allows managers to ask questions to gain insight into what their employees think about work, the culture, the direction of the company, what people are working on, or even something personal like their favorite recent movie. We created KYC because when the Basecamp team grew to about 30 people, Jason realized he didn't know the company or its people as well as he should have. So, KYC was created initially as an internal tool that eventually became a product that we sold to customers. In 2014, we [sold Know Your Company](https://knowyourcompany.com/story) to Claire Lew, and she's been running it ever since!
+Ten years ago, to-do list apps were few and far between. So, we created Ta-Da List as a stand alone free subscription app at tadalist.com in January 2005. Now that people have to-do lists on their phones, desktops, laptops, tablets, and refrigerator doors, we recognized that Ta-Da List was a little outdated. We shuttered tadalist.com in 2014, but existing customers were able to retain their accounts.
```
Comment (dhh) [inline product-histories.md]: Think we should be clear that for us "shuttered" always simply means "closed for new customers". Sounds like it stopped working.

File: product-histories.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Product Histories
 
+For a concise summary of this section, see the tabular data of our products, when they were launched, and when they "retired" [here](https://github.com/basecamp/data_scratchpad/blob/master/miscellany/data/product_dates.csv).
```
Comment (dhh) [inline product-histories.md]: This handbook is public, so we try mostly not to link to other data sources that are private, when we can avoid it. I would either nix this or put a summary table in right here that's a copy of that.

File: product-histories.md [hunk 3]
```diff
@@ -16,32 +18,40 @@ Fast forward to 2014. We'd been feeling pulled in too many directions for too lo
...
+Basecamp has run a job board since 2006, helping connect small businesses with good designers and programmers for full-time jobs, freelance gigs, and internships.
+
+In 2014, Basecamp revamped our job board and launched We Work Remotely. After Jason and David published their book REMOTE, we realized we wanted to be on the forefront of promoting remote work. So, we created and hosted WeWorkRemotely.com, which contains job postings for programmers, designers, marketing professionals, customer service representatives, writers, and more -- all remote work positions. To post a job, a company needs to fill out a form on our site with information about the job, pay Basecamp $200 for a 30 day posting, and then verify that they are in fact offering a remote job. We've posted over 3,500 jobs over the last few years. In Spring 2017, we sold We Work Remotely to Metalab, to cut down on manual labor taken to run the board and focus more on our core business - Basecamp!
+
+## Sortfolio.com
```
Comment (dhh) [inline product-histories.md]: Add return.

**Comment by janejuenyang** [comment]
Thanks for the review, David! Made changes based on your feedback.

**Pull request merged**